src/content/reference/react-dom/hydrate.md

```diff
@@ -24,59 +24,59 @@ hydrate(reactNode, domNode, callback?)
-* You'll likely have only one `hydrate` call in your app. If you use a framework, it might do this call for you.
-* If your app is client-rendered with no HTML rendered already, using `hydrate()` is not supported. Use [render()](/reference/react-dom/render) (for React 17 and below) or [createRoot()](/reference/react-dom/client/createRoot) (for React 18+) instead.
+#### Catatan {/*caveats*/}
+* `hydrate` mengharapkan konten yang di-*render* identik dengan konten yang di-*render* di server. React dapat memperbaiki perbedaan dalam konten teks, tetapi anda harus memperlakukan ketidakcocokan sebagai bug dan memperbaikinya.
+* Dalam mode pengembangan, React memberi peringatan tentang ketidakcocokan selama hydration. Tidak ada jaminan bahwa perbedaan atribut akan diperbaiki dalam kasus ketidakcocokan. Ini penting untuk alasan kinerja karena dalam sebagian besar aplikasi, ketidakcocokan jarang terjadi, sehingga memvalidasi semua markup akan menjadi sangat mahal.
+* Kemungkinan anda hanya akan memiliki satu `hydrate` panggilan dalam aplikasi anda. Jika anda menggunakan sebuah *framework*, mungkin *framework* tersebut melakukan panggilan ini untuk anda.
+* Jika aplikasi anda di-*render* oleh klien tanpa HTML yang sudah di-*render* sebelumnya, menggunakan `hydrate()` tidak didukung. Gunakan [render()](/reference/react-dom/render) (untuk React 17 dan lebih rendah) or [createRoot()](/reference/react-dom/client/createRoot) (untuk React 18+) sebaliknya.
 
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-Call `hydrate` to attach a <CodeStep step={1}>React component</CodeStep> into a server-rendered <CodeStep step={2}>browser DOM node</CodeStep>.
+Panggil `hydrate` untuk melekatkan sebuah <CodeStep step={1}>React component</CodeStep> menjadi server-rendered <CodeStep step={2}>peramban DOM node</CodeStep>.
 
 ```js [[1, 3, "<App />"], [2, 3, "document.getElementById('root')"]]
 import { hydrate } from 'react-dom';
 
 hydrate(<App />, document.getElementById('root'));
 ````
 
-Using `hydrate()` to render a client-only app (an app without server-rendered HTML) is not supported. Use [`render()`](/reference/react-dom/render) (in React 17 and below) or [`createRoot()`](/reference/react-dom/client/createRoot) (in React 18+) instead.
+Menggunakan `hydrate()` untuk me-*render* aplikasi hanya di sisi klien (aplikasi tanpa HTML yang di-*render* oleh server) tidak didukung. Gunakan [`render()`](/reference/react-dom/render) (di React 17 dan lebih rendah) atau [`createRoot()`](/reference/react-dom/client/createRoot) (di React 18+) sebaliknya.
 
-### Hydrating server-rendered HTML {/*hydrating-server-rendered-html*/}
+### Hydrating HTML yang di-*render* oleh server {/*hydrating-server-rendered-html*/}
```

Comment:
```suggestion
### Menghidrasi HTML yang di-*render* oleh server {/*hydrating-server-rendered-html*/}
```

If we agree to translate "hydration" to "*hidrasi*", then "hydrating" can be naturally translated to "*menghidrasi*".